commitlint.config.js

```diff
@@ -0,0 +1 @@
+module.exports = { extends: ['@commitlint/config-conventional'] };
```

Comment:
Taken right from [the local setup guide](https://commitlint.js.org/#/guides-local-setup).

Here are the [rules](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional).